### PR TITLE
Release v0.1

### PR DESCRIPTION
**Pull request by ericpre**
Reference: https://github.com/hyperspy/rosettasciio/blob/main/releasing_guide.md

If other PR get merged in the mean time, I will rebase and run `trowncrier build` another time.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -8,6 +8,94 @@ https://rosettasciio.readthedocs.io/en/latest/changes.html
...
+================
+
+New features
+------------
+
+- Add support for reading the ``.xml``-format from Horiba Jobin Yvon's LabSpec software. (`#25 <https://github.com/hyperspy/rosettasciio/issues/25>`_)
+- Add support for reading the ``.tvf``-format from TriVista. (`#27 <https://github.com/hyperspy/rosettasciio/issues/27>`_)
+- Add support for reading the ``.wdf``-format from Renishaw's WIRE software. (`#55 <https://github.com/hyperspy/rosettasciio/issues/55>`_)
+- Added subclassing of .sur files in CL signal type and updated metadata parsing (`#98 <https://github.com/hyperspy/rosettasciio/issues/98>`_)
+- add optional kwarg to tiff reader ``multipage_as_list`` which when set to True uses ``pages`` interface and returns list of signal for every page with full metadata. (`#104 <https://github.com/hyperspy/rosettasciio/issues/104>`_)
```
Comment (jlaehne) [inline CHANGES.rst:21]:
```suggestion
- Add optional kwarg to tiff reader ``multipage_as_list`` which when set to True uses ``pages`` interface and returns list of signal for every page with full metadata. (`#104 <https://github.com/hyperspy/rosettasciio/issues/104>`_)
```

**Review by jlaehne — APPROVED** [review]

**Review by jlaehne — APPROVED** [review]

**Comment by ericpre** [comment]
https://pypi.org/project/rosettasciio 😃

**Comment by francisco-dlp** [comment]
For some reason the zenodo record doesn't include the authors.

**Comment by ericpre** [comment]
I have updated the entry manually, following https://github.com/hyperspy/rosettasciio/graphs/contributors as we would have expected to be done automatically. The "Cite As" still says github-actions[bot]. I have opened https://github.com/zenodo/zenodo/issues/2462.

**Pull request merged**